ci/build/build-standalone-release.sh

```diff
@@ -28,7 +28,7 @@ main() {
   ln -s "./lib/node" "$RELEASE_PATH/node"
 
   pushd "$RELEASE_PATH"
-  yarn --production --frozen-lockfile
+  npm install --unsafe-perm --omit=dev
```

Comment:
That'll help us catch this bug in the future.

Reply:
Sorry I didn't get to do this yet... It was on my to-do list from https://github.com/coder/code-server/pull/5071#discussion_r950597128 but hadn't gotten to it yet...

Reply:
Oh don't worry! All good 👍🏼 We didn't realize it would prevent other things so no worries at all.

Reply:
I vaguely recall some issue with `npm ci` and optional dependencies; does that ring any bells?

Reply:
I think those were fixed with the latest versions of NPM, namely NPM 8 which we have since we bumped to NodeJS 16?

Reply:
Ahh gotcha!